com.unity.multiplayer.mlapi/Editor/CodeGen/ILPostProcessorProgram.cs

```diff
@@ -11,8 +11,31 @@
 
 using Assembly = System.Reflection.Assembly;
 
+using ILPPInterface = Unity.CompilationPipeline.Common.ILPostProcessing.ILPostProcessor;
+
 namespace MLAPI.Editor.CodeGen
 {
+    // There is a behaviour difference between 2019.4 and 2020+ codegen
+    // that essentially does checking on the existence of ILPP vs if a CodeGen assembly
+    // is present. So in order to make sure ILPP runs properly in 2019.4 from a clean
+    // import of the project we add this dummy ILPP which forces the callback to made
+    // and meets the internal ScriptCompilation pipeline requirements
+    internal sealed class HackILPPToMakeThingsWork : ILPPInterface
```

Comment:
I recommend against using the word "Hack" in any method name.

Reply:
+1, let me try another name here.

```suggestion
    internal sealed class DummyILPP : ILPPInterface
```

Reply:
`DefaultILPP` would also look nice too!

Reply:
I like Default the best

Reply:
I'd like to go with `ILPP2019CodegenWorkaround`

Reply:
sounds good to me!